.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -13,14 +13,27 @@
 
 .upload-entities-to-cortex-xsoar: &upload-entities-to-cortex-xsoar
   - section_start "Upload Entities to Cortex XSOAR" --collapsed
-  - demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure
   - demisto-sdk upload -i Packs/HelloWorld/TestPlaybooks/playbook-HelloWorld-Test.yml --insecure
-  - demisto-sdk upload -i Packs/HelloWorld/Layouts/layoutscontainer-Hello_World_Test_Layout.json --insecure
   - demisto-sdk upload -i Packs/HelloWorld/IncidentFields/incidentfield-Hello_World_IncidentField_Test.json --insecure
   - demisto-sdk upload -i Packs/HelloWorld/IncidentTypes/incidenttype-Hello_World_Alert_Test.json --insecure
-  - demisto-sdk upload -i Packs/HelloWorld/Classifiers/classifier-mapper-incoming-HelloWorldTest.json --insecure
   - section_end "Upload Entities to Cortex XSOAR"
 
+
+.export_cloud_machine_constants: &export_cloud_machine_constants
+  - CLOUD_SERVERS_PATH=$(cat $CLOUD_SERVERS_FILE)
+  - echo ${CLOUD_API_KEYS} > "cloud_api_keys.json"
+  - echo "${CLOUD_API_TOKENS}" > "cloud_api_tokens.json"
```

Comment:
@michal-dagan please guide @MLainer1 on the required changes here if relevant

Reply:
thanks - @MLainer1 please adjust as explained here: https://github.com/demisto/content/pull/30367